build.gradle

```diff
@@ -12,8 +12,8 @@ war {
   archiveName = 'radar.war'
 }
 
-targetCompatibility = '1.7'
-sourceCompatibility = '1.7'
+targetCompatibility = '1.8'
+sourceCompatibility = '1.8'
```

Comment:
I've updated the Tomcat image to use Java 8 in RADAR-CNS/RADAR-Docker@e8f1f99, I suppose that would solve the issue?